src/background/index.mjs

```
@@ -52,219 +52,427 @@ import { generateAnswersWithMoonshotWebApi } from '../services/apis/moonshot-web
+  console.log(
+    `[background] executeApi called for model: ${session.modelName}, apiMode: ${session.apiMode}`,
+  )
+  console.debug('[background] Full session details:', session)
+  // Redact sensitive config details before logging
+  const redactedConfig = { ...config };
+  if (redactedConfig.apiKey) redactedConfig.apiKey = 'REDACTED';
+  if (redactedConfig.customApiKey) redactedConfig.customApiKey = 'REDACTED';
+  if (redactedConfig.claudeApiKey) redactedConfig.claudeApiKey = 'REDACTED';
+  if (redactedConfig.kimiMoonShotRefreshToken) redactedConfig.kimiMoonShotRefreshToken = 'REDACTED';
+  // Add any other sensitive keys that might exist in 'config' or 'session.apiMode'
+  if (session.apiMode && session.apiMode.apiKey) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)